robot/EDA/resources/ContactsPageObject.py

```diff
@@ -313,8 +313,8 @@ def Clear_the_disable_preferred_phone_enforcement(self):
         )
 
     def Run_phone_cleanup(self):
-        """ Click on the 'Run Phone Cleanup' button.
-            Also verifies that the 'Run Phone Cleanup' was queued to run.
+        """ Click on the 'Run Cleanup' button.
```

Comment:
@dashattuck It looks like you made this change -- is this just updating the comment, and so should be fine to keep here? Not sure if you meant to put the change here or in a separate Robot branch.